README.md

```diff
@@ -71,7 +71,7 @@ https://jwst-pipeline.readthedocs.io/en/latest/
 
 One can clone this repository and build the documentation with
 
-    pip install sphinx_rtd_theme stsci_rtd_theme sphinx_automodapi
+    pip install sphinx_rtd_theme stsci_rtd_theme sphinx_automodapi sphinxcontrib_programoutput
```

Comment:
I don't see `sphinxcontrib_programoutput` directive used anywhere in this PR. I guess you know best, although I am concerned that usage of this extension would impede any doctest that you might have.

Reply:
@nden added it in #2955 and I found out the hard way that it's now required in order to `build_sphinx`.